pkg/envoy/xds/cache.go

```diff
@@ -254,3 +254,8 @@ func (c *Cache) GetResources(ctx context.Context, typeURL string, lastVersion *u
 	cacheLog.Debugf("returning %d resources out of %d requested", len(res.Resources), len(resourceNames))
 	return res, nil
 }
+
+// GetVersion returns the version for the cache at the current point in time.
+func (c *Cache) GetVersion() uint64 {
```

Comment:
Remove. That seems unused.
Anyway it is not thread-safe as-is. This would require locking.